Snakefile

```diff
@@ -548,3 +473,96 @@ rule clean:
     run:
         shell("rm -r ../pypsa-earth/resources")
         shell("rm -r ../pypsa-earth/networks")
+
+
+if config["custom_data"].get("industry_demand", False) == True:
```

Comment:
It would shorten the Snakefile a lot if you avoid an if-statement here but include it within the input/output files

Reply:
The thing is, some rules are not needed in case of custom data. Rule build_industrial_production_tomorrow is skipped.
I will open this as an enhancement issue because I think it will take some time.

Reply:
True, something worth looking at once it is merged with PyPSA-Earth I guess